docs/how-to-guides/access-monitoring.md

```diff
@@ -0,0 +1,81 @@
+# Access Monitoring on Lokomotive
```

Comment:
As said before, I feel like "how to open Grafana in my browser" is too narrow a topic for a self-contained tutorial :-) I feel like this material should appear at the end of a more complete tutorial which explains how to deploy the Lokomotive monitoring stack. To emphasize, the config reference isn't a tutorial as it doesn't explain **how to do things**.

Since we currently don't seem to have such a tutorial, I suggest we create a "skeleton" for such a tutorial with `TODO` in all the sections you haven't written, include your new sections there and open an issue for completing the guide. You can use [this](https://github.com/kinvolk/lokomotive/blob/master/docs/how-to-guides/authentication-with-dex-gangway.md) how-to guide as an example if it helps you.

Reply:
I have  added information about how to deploy, PTAL.